test/test_month_to_season.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+from geocat.comp.month_to_season import month_to_season
+
+
+def get_fake_dataset(start_month, nmonths, nlats, nlons):
+    """ Returns a very simple xarray dataset for testing.
+        Data values are equal to "month of year" for monthly time steps.
+    """
+    # Create coordinates
+    months = pd.date_range(start=pd.to_datetime(start_month), periods=nmonths, freq='MS')
+    lats = np.linspace(start=-90, stop=90, num=nlats, dtype='float32')
+    lons = np.linspace(start=-180, stop=180, num=nlons, dtype='float32')
+
+    # Create data variable. Construct a 3D array with time as the first dimension.
+    month_values = np.expand_dims(np.arange(start=1, stop=nmonths + 1), axis=(1, 2))
+    var_values = np.tile(month_values, (1, nlats, nlons))
+
+    ds = xr.Dataset(
+        data_vars={
+            'my_var': (('time', 'lat', 'lon'), var_values.astype('float32')),
+        },
+        coords={'time': months, 'lat': lats, 'lon': lons},
+    )
+    return ds
+
+
+class Test_month_to_season(unittest.TestCase):
+
+    def setUp(self):
+        # Create a dataset for the year 2000.
+        self.ds1 = get_fake_dataset(start_month='2000-01', nmonths=12, nlats=1, nlons=1)
+
+        # Create another dataset for the year 2001.
+        self.ds2 = get_fake_dataset(start_month='2001-01', nmonths=12, nlats=1, nlons=1)
+
+        # Create a dataset that combines the two previous datasets, for two years of data.
+        self.ds3 = xr.concat([self.ds1, self.ds2], dim='time')
+
+        # Create a dataset with the wrong number of months.
+        self.partial_year_dataset = get_fake_dataset(start_month='2000-01', nmonths=13, nlats=1, nlons=1)
+
+        # Create a dataset with a custom time coordinate.
+        custom_time_dataset = get_fake_dataset(start_month='2000-01', nmonths=12, nlats=1, nlons=1)
+        self.custom_time_dataset = custom_time_dataset.rename_dims({'time': 'my_time'})
```

Comment:
```suggestion
        self.custom_time_dataset = custom_time_dataset.rename({'time': 'my_time'})
```

This is what is causing the test to fail. As the name hints, this will rename dimension names only. What you want is to rename the time coordinate as well. 

```python
In [20]: ds.rename_dims({'time': 'my_time'})
Out[20]:
<xarray.Dataset>
Dimensions:  (lat: 1, lon: 1, my_time: 12)
Coordinates:
    time     (my_time) datetime64[ns] 2000-01-01 2000-02-01 ... 2000-12-01
  * lat      (lat) float32 -90.0
  * lon      (lon) float32 -180.0
Dimensions without coordinates: my_time
Data variables:
    my_var   (my_time, lat, lon) float32 1.0 2.0 3.0 4.0 ... 9.0 10.0 11.0 12.0
```

```python
In [21]: ds.rename({'time': 'my_time'})
Out[21]:
<xarray.Dataset>
Dimensions:  (lat: 1, lon: 1, my_time: 12)
Coordinates:
  * my_time  (my_time) datetime64[ns] 2000-01-01 2000-02-01 ... 2000-12-01
  * lat      (lat) float32 -90.0
  * lon      (lon) float32 -180.0
Data variables:
    my_var   (my_time, lat, lon) float32 1.0 2.0 3.0 4.0 ... 9.0 10.0 11.0 12.0
```

Reply:
Thank you again, Anderson!   Unit tests are all passing now.